crates/gitbutler-tauri/src/open.rs

```diff
@@ -0,0 +1,71 @@
+use crate::error::Error;
+use anyhow::anyhow;
+use std::env;
+use tracing::instrument;
+use url::Url;
+
+pub(crate) fn open_that(path: &str) -> Result<(), Error> {
+    let target_url = match Url::parse(path) {
+        Ok(url) => url,
+        Err(_) => return Err(anyhow!("Invalid path format").into()),
+    };
```

Comment:
```suggestion
    let target_url = Url::parse(path).with_context(|| format!("Invalid path format: '{path}'")?;
```

Reply:
I tried it initially with the question mark syntax, but it complained about the potnential error not matching some frontend error type, is that avoided by adding some context here?

Reply:
Right, that reminds me of another issue regarding the error types used here.
I have pushed a fix for my recent suggestions, leaving only the error handling part.

Reply:
Great, thanks!